cmd/ddev/cmd/remove.go

```diff
@@ -14,13 +18,20 @@ var LocalDevRMCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		app, err := getActiveApp()
 		if err != nil {
-			util.Failed("Failed to remove: %v", err)
+			util.Failed("Failed to get active app: %v", err)
 		}
 
 		if app.SiteStatus() == "not found" {
 			util.Failed("App not running locally. Try `ddev start`.")
 		}
 
+		if !skipConfirmation {
+			fmt.Printf("Is it ok to remove the site %s with all of its containers? All data will be lost. (y/N): ", app.GetName())
+			if !util.AskForConfirmation() {
+				util.Failed("Not removing site.")
```

Comment:
I found this message a bit jarring after cancelling the removal. My first thought was that I had done something wrong. Perhaps it would be better as a yellow warning text? "Site removal cancelled" might be a bit clearer as well.